src/events/eventActions.js

```diff
@@ -1,97 +1,94 @@
 /* @flow */
-import type { Auth, Dispatch, GetState, Action } from '../types';
+import type { Dispatch, GetState, Action } from '../types';
 import { pollForEvents, registerForEvents } from '../api';
 import { switchAccount } from '../account/accountActions';
 import { timeout } from '../utils/async';
 import eventToAction from './eventToAction';
 import eventMiddleware from './eventMiddleware';
+import { getAuth } from '../account/accountSelectors';
 
-import {
-  BATCH_ACTIONS,
-  REALM_INIT,
-  EVENT_REGISTERED,
-} from '../actionConstants';
+import { BATCH_ACTIONS, REALM_INIT, EVENT_REGISTERED } from '../actionConstants';
 
-const startEventPolling = (auth: Auth, queueId: number, eventId: number) =>
-  async (dispatch: Dispatch, getState: GetState) => {
-    let lastEventId = eventId;
-    let numFailures = 0;
+const startEventPolling = (queueId: number, eventId: number) => async (
```

Comment:
I agree that this is not consistent. I used latest atom prettier plugin for formatting. I guess [this](https://prettier.io/playground/#%7B%22content%22%3A%22%2F*%20%40flow%20*%2F%5Cnimport%20type%20%7B%20Auth%2C%20Dispatch%2C%20GetState%2C%20Action%20%7D%20from%20'..%2Ftypes'%3B%5Cnimport%20%7B%20pollForEvents%2C%20registerForEvents%20%7D%20from%20'..%2Fapi'%3B%5Cnimport%20%7B%20switchAccount%20%7D%20from%20'..%2Faccount%2FaccountActions'%3B%5Cnimport%20%7B%20timeout%20%7D%20from%20'..%2Futils%2Fasync'%3B%5Cnimport%20eventToAction%20from%20'.%2FeventToAction'%3B%5Cnimport%20eventMiddleware%20from%20'.%2FeventMiddleware'%3B%5Cn%5Cnimport%20%7B%5Cn%20%20BATCH_ACTIONS%2C%5Cn%20%20REALM_INIT%2C%5Cn%20%20EVENT_REGISTERED%2C%5Cn%7D%20from%20'..%2FactionConstants'%3B%5Cn%5Cnconst%20startEventPolling%20%3D%20(auth%3A%20Auth%2C%20queueId%3A%20number%2C%20eventId%3A%20number)%20%3D%3E%5Cn%20%20async%20(dispatch%3A%20Dispatch%2C%20getState%3A%20GetState)%20%3D%3E%20%7B%5Cn%20%20%20%20let%20lastEventId%20%3D%20eventId%3B%5Cn%20%20%20%20let%20numFailures%20%3D%200%3B%5Cn%5Cn%20%20%20%20%2F%2F%20Event%20loop%5Cn%20%20%20%20%2F%2F%20eslint-disable-next-line%20no-constant-condition%5Cn%20%20%20%20while%20(true)%20%7B%5Cn%20%20%20%20%20%20let%20response%3B%5Cn%20%20%20%20%20%20try%20%7B%5Cn%20%20%20%20%20%20%20%20%2F%2F%20eslint-disable-next-line%20no-await-in-loop%5Cn%20%20%20%20%20%20%20%20response%20%3D%20await%20pollForEvents(auth%2C%20queueId%2C%20lastEventId)%3B%5Cn%20%20%20%20%20%20%7D%20catch%20(e)%20%7B%5Cn%20%20%20%20%20%20%20%20%2F%2F%20Stop%20polling%20-%20user%20likely%20switched%20accounts%20or%20logged%20out%5Cn%20%20%20%20%20%20%20%20if%20(queueId%20!%3D%3D%20getState().app.eventQueueId)%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20break%3B%5Cn%20%20%20%20%20%20%20%20%7D%5Cn%5Cn%20%20%20%20%20%20%20%20%2F%2F%20Force%20a%20refresh%20of%20the%20app%20if%20the%20event%20queue%20is%20too%20old%5Cn%20%20%20%20%20%20%20%20%2F%2F%20or%20has%20been%20garbage%20collected%5Cn%20%20%20%20%20%20%20%20if%20(e.message.indexOf('too%20old')%20!%3D%3D%20-1%20%7C%7C%5Cn%20%20%20%20%20%20%20%20%20%20%20%20e.message.indexOf('Bad%20event%20queue%20id')%20!%3D%3D%20-1)%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20%2F%2F%20Force%20a%20refresh%20(index%200%20is%20the%20currently%20active%20account)%5Cn%20%20%20%20%20%20%20%20%20%20dispatch(switchAccount(0))%3B%5Cn%20%20%20%20%20%20%20%20%20%20break%3B%5Cn%20%20%20%20%20%20%20%20%7D%5Cn%5Cn%20%20%20%20%20%20%20%20%2F%2F%20Use%20exponential%20back-off%20when%20retrying%5Cn%20%20%20%20%20%20%20%20const%20retrySec%20%3D%20Math.min(90%2C%20Math.exp(numFailures%20%2F%202))%3B%5Cn%20%20%20%20%20%20%20%20%2F%2F%20eslint-disable-next-line%20no-await-in-loop%5Cn%20%20%20%20%20%20%20%20await%20timeout(retrySec%20*%201000)%3B%5Cn%20%20%20%20%20%20%20%20numFailures%2B%2B%3B%5Cn%20%20%20%20%20%20%20%20continue%3B%20%2F%2F%20eslint-disable-line%20no-continue%5Cn%20%20%20%20%20%20%7D%5Cn%20%20%20%20%20%20%2F%2F%20Stop%20polling%20-%20user%20likely%20switched%20accounts%20or%20logged%20out%5Cn%20%20%20%20%20%20if%20(queueId%20!%3D%3D%20getState().app.eventQueueId)%20%7B%5Cn%20%20%20%20%20%20%20%20break%3B%5Cn%20%20%20%20%20%20%7D%5Cn%5Cn%20%20%20%20%20%20numFailures%20%3D%200%3B%5Cn%5Cn%20%20%20%20%20%20const%20actions%20%3D%20%5B%5D%3B%5Cn%20%20%20%20%20%20for%20(const%20event%20of%20response.events)%20%7B%5Cn%20%20%20%20%20%20%20%20lastEventId%20%3D%20Math.max(lastEventId%2C%20event.id)%3B%5Cn%20%20%20%20%20%20%20%20eventMiddleware(getState()%2C%20event)%3B%5Cn%20%20%20%20%20%20%20%20const%20action%20%3D%20eventToAction(getState()%2C%20event)%3B%5Cn%5Cn%20%20%20%20%20%20%20%20if%20(action)%20%7B%5Cn%20%20%20%20%20%20%20%20%20%20actions.push(%7B%20...action%2C%20eventId%3A%20event.id%20%7D)%3B%5Cn%20%20%20%20%20%20%20%20%7D%5Cn%20%20%20%20%20%20%7D%5Cn%5Cn%20%20%20%20%20%20if%20(actions.length%20%3E%201)%20%7B%5Cn%20%20%20%20%20%20%20%20%2F%2F%20Batch%20actions%20together%20to%20speed%20up%20rendering%5Cn%20%20%20%20%20%20%20%20%2F%2F%20(especially%20when%20resuming%20from%20a%20suspended%20state)%5Cn%20%20%20%20%20%20%20%20dispatch(%7B%20type%3A%20BATCH_ACTIONS%2C%20actions%20%7D)%3B%5Cn%20%20%20%20%20%20%7D%20else%20if%20(actions.length%20%3D%3D%3D%201)%20%7B%5Cn%20%20%20%20%20%20%20%20dispatch(actions%5B0%5D)%3B%5Cn%20%20%20%20%20%20%7D%5Cn%20%20%20%20%7D%5Cn%20%20%7D%3B%5Cn%5Cnexport%20const%20fetchEvents%20%3D%20(auth%3A%20Auth)%3A%20Action%20%3D%3E%5Cn%20%20async%20(dispatch%3A%20Dispatch%2C%20getState%3A%20GetState)%20%3D%3E%20%7B%5Cn%20%20%20%20const%20data%20%3D%20await%20registerForEvents(auth)%3B%5Cn%5Cn%20%20%20%20const%20queueId%20%3D%20data.queue_id%3B%5Cn%20%20%20%20const%20lastEventId%20%3D%20data.last_event_id%3B%5Cn%5Cn%20%20%20%20dispatch(%7B%5Cn%20%20%20%20%20%20type%3A%20REALM_INIT%2C%5Cn%20%20%20%20%20%20data%2C%5Cn%20%20%20%20%7D)%3B%5Cn%5Cn%20%20%20%20dispatch(%7B%5Cn%20%20%20%20%20%20type%3A%20EVENT_REGISTERED%2C%5Cn%20%20%20%20%20%20queueId%2C%5Cn%20%20%20%20%20%20lastEventId%2C%5Cn%20%20%20%20%7D)%3B%5Cn%5Cn%20%20%20%20%2F%2F%20Start%20the%20event%20loop%5Cn%20%20%20%20dispatch(startEventPolling(auth%2C%20queueId%2C%20lastEventId))%3B%5Cn%20%20%7D%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Atrue%2C%22trailingComma%22%3A%22all%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D) is what prettier.js formatting does by default in this case.